Doxyfile

```diff
@@ -816,7 +816,7 @@ EXCLUDE_PATTERNS       =
 # Note that the wildcards are matched against the file with absolute path, so to
 # exclude all test directories use the pattern */test/*
 
-EXCLUDE_SYMBOLS        =
+EXCLUDE_SYMBOLS        = MappingTests MeshTests CplSchemeTests PreciceTests
```

Comment:
Why are them excluded? Why only these test suite? Why are tests at all excluded?

Reply:
These are the symbols generated by Boost.Test and are only mentioned as forward-, and friend-declarations in our code.
So, they are currently listed as empty entries under [namespaces](http://xgm.de/precice/docs/develop/namespaceCplSchemeTests.html).

Looking at this again I realise that this will not scale in the future.
Maybe it would be better to wrap the fwd/friend declarations in `#ifndef DOXYGEN_SHOULD_SKIP_THIS` blocks.

Reply:
Ok, now I understand.

I think blacklisting them in the Doxyfile is fine then. Don't like the `#ifndef` stuff, would just clutter the code and the Doxyfile is read only seldom, so we should move clutter there.

Reply:
I don't like it either. Maybe we can create a single friend helper class to avoid all this clutter.